Comment afficher la 4ème de couverture de mon livre.

joannagb opened the pull request:
Pour cette recherche utilisateur avec mots clés ' 4ème couverture'" j'ai pensé aux questions du type : 
1.  Est-il possible d'ajouter une 4ème couverture à la page d'accueil ?" 
2.  Est-ce possible d'ajouter d'autres couvertures aux 'Livres récents' (de la page d'acceuil) ?  
un mix des deux ? Autre chose ? Qu'en pensez-vous ?
-----------------
question mise dans la section : FAQ Liseuse
-----------------
Réponse  : Le module des  'Livres récents' permet d'afficher les couvertures de trois derniers livres ouverts.  Actuellement, il n'est pas possible d'en ajouter d'autres./ d'en ajouter une 4ème.

joannagb committed:
Update FAQ-liseuse.md

Pour cette recherche utilisateur avec mots clés ' 4ème couverture'" j'ai trouvé comme possibilités de question : 
1. Est-il possible d'ajouter une 4ème couverture à la page d'accueil ?"
2. Est-ce possible d'ajouter d'autres couvertures aux  'Livres récents' (de la page d'acceuil) ?  
un mix des deux ? Autre chose ? Qu'en pensez-vous ?
Section FAQ Liseuse

fsechaud commented:
Il s'agit peut-être de pouvoir afficher la 4ème de couverture. C'est une demande qui revenait régulièrement dans l'enquête de satisfaction.

astranchet commented:
@joannagb : plutôt d'accord avec @fsechaud : la demande est généralement "comment afficher la 4ème de couverture de mon livre". La réponse est que hélas, contrairement au livre papier, le livre numérique n'en a pas (tu peux lire cet article intéressant : https://page42.org/la-quatri-me-de-couverture-grande-absente-du-livre/) On peut cependant quand même dire qu'on peut afficher plus d'informations sur le livre via un appui long sur une couverture puis choisir "Informations sur le livre" par exemple.

joannagb commented:
Merci à toutes les deux , je corrige.

joannagb committed:
4ème de couverture

modification de la réponse  pour vérif et correction
##### Comment afficher la 4ème de couverture de mon livre ?

La réponse est que hélas, contrairement au livre papier, le livre numérique n'en a pas. Cependant,  vous pouvez, via un appui long sur une couverture, trouver la plupart d'éléments de la 4ème de couverture dans les "Informations sur le livre".

joannagb committed:
modification texte

Proposition du texte : Le livre numérique ne possède pas la 4ème de couverture.  Toutefois, certaines de ses informations ont été reprises dans la section "Infos sur le livre".  Pour les consulter : faites un appui long sur une couverture puis sélectionnez : "Infos sur le livre"

astranchet committed:
Simplifie la réponse
